CartoDB/cartodb
# Fixing problems with custom color and image infowindow templates

It will fix https://github.com/CartoDB/cartodb/issues/10975.

**Steps to test**
- Create a map of points with several columns.
- Add a click popup and select the color one.
- Select several columns.
- Open a popup in order to see it.
- Choose html and check that the html is representing the same thing you chose in the form (visit https://github.com/CartoDB/cartodb/issues/10975 in order to check differences).
- Same for image 💋 .
